<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Delvian Valentine</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yocto Keys</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>18</number>
    </property>
    <property name="topMargin">
     <number>18</number>
    </property>
    <property name="rightMargin">
     <number>18</number>
    </property>
    <property name="bottomMargin">
     <number>18</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>User</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="edit_user">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_sign">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Sign</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Licence key</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="edit_licence_key">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>424</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKeys">
    <property name="title">
     <string>Keys</string>
    </property>
    <addaction name="action_new_rsa_keys"/>
    <addaction name="action_priv_key"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuKeys"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_about">
   <property name="text">
    <string>About Yocto Keys</string>
   </property>
  </action>
  <action name="action_new_rsa_keys">
   <property name="text">
    <string>New RSA Keysâ€¦</string>
   </property>
  </action>
  <action name="action_priv_key">
   <property name="text">
    <string>Private Key</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>edit_user</sender>
   <signal>returnPressed()</signal>
   <receiver>button_sign</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
